tests/functional/centos/7/cluster/ceph-override.json

```diff
@@ -2,7 +2,10 @@
 	"ceph_conf_overrides": {
 		"global": {
 			"osd_pool_default_pg_num": 12,
-			"osd_pool_default_size": 1
+			"openstack_cephfs_metadata_pool": {
```

Comment:
This must be outside the `ceph_conf_overrides` dict.
Because of that your code is never triggered.

Reply:
Done.